application/models/Survey.php

```diff
@@ -1256,4 +1257,31 @@ public function getsSurveyUrl()
         return $this->sSurveyUrl;
     }
 
+    /**
+     * @return Question[]
+     */
+    public function getQuotableQuestions()
+    {
+        $criteria=new CDbCriteria;
+        $criteria->select = 't.*';
+        $criteria->with=array(
+            'survey.groups',
+        );
+
+        $criteria->order ='`groups`.`group_order`, `t`.`question_order`';
```

Comment:
@Shnoulle Thanks!
No, tested ONLY on Msql. 
I added the quoteColumnName as you suggested. 
I do not have the mssql testing option right now.